zilliqa/src/cfg.rs

```diff
@@ -547,6 +548,10 @@ pub struct ForkDelta {
     /// exempt addresses.
     #[serde(default)]
     pub scilla_call_gas_exempt_addrs: Vec<Address>,
+
+    /// If true, the total gas fee for all transactions in a block will be transferred to the zero account.
+    /// Before this fork, we were incorrectly transferring the gas used by all transactions to the zero account.
```

Comment:
```suggestion
    /// If true, the total gas paid by all transactions in a block is transferred to the zero address. This keeps the total supply of the network constant. If false, we still transfer funds to the zero address, but with an incorrect gas price of 1 Wei per gas.
```

minor wording changes - take it or leave it :)